### PR TITLE
Corrected Grammatical Errors.

### LINKED ISSUE #12300: Grammatical
This is not really a bug, but I don't know where else to point it out.

The motorBike checklist should read "an engines attribute"  not "a engines attribute"

Challenge [Declare JavaScript Objects as Variables](https://www.freecodecamp.com/challenges/declare-javascript-objects-as-variables#?solution=%0Avar%20car%20%3D%20%7B%0A%20%20%22wheels%22%3A4%2C%0A%20%20%22engines%22%3A1%2C%0A%20%20%22seats%22%3A5%0A%7D%3B%0A%0Avar%20motorBike%20%3D%20%7B%0A%0A%0A%20%20%2F%2F%20Only%20change%20code%20below%20this%20line.%0A%20%20%22wheels%22%3A2%2C%0A%20%20%22engines%22%3A1%2C%0A%20%20%22seats%22%3A1%0A%7D%3B%0A) has an issue.
User Agent is: <code>Mozilla/5.0 (Windows NT 10.0; WOW64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/55.0.2883.87 Safari/537.36</code>.
Please describe how to reproduce this issue, and include links to screenshots if possible.

My code:
```javascript

var car = {
  "wheels":4,
  "engines":1,
  "seats":5
};

var motorBike = {


  // Only change code below this line.
  "wheels":2,
  "engines":1,
  "seats":1
};

```

### PR DESCRIPTION
**Pull request by 91aman**
#### Pre-Submission Checklist

- [x] Your pull request targets the `staging` branch of FreeCodeCamp.
- [x] Branch starts with either `fix/`, `feature/`, or `translate/` (e.g. `fix/signin-issue`)
- [x] You have only one commit (if not, [squash](http://forum.freecodecamp.com/t/how-to-squash-multiple-commits-into-one-with-git/13231) them into one commit).
- [x] All new and existing tests pass the command `npm test`. Use `git commit --amend` to amend any fixes.

#### Type of Change

- [x] Small bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds new functionality)
- [ ] Breaking change (fix or feature that would change existing functionality)
- [ ] Add new translation (feature adding new translations)

#### Checklist:

- [x] Tested changes locally.
- [x] Closes currently open issue (replace XXXX with an issue no): Closes #12300

#### Description
Corrected the grammar changes as pointed out in the issue no #12300

### CODE REVIEW

File: seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json [hunk 1]
```diff
@@ -18,7 +18,7 @@
       "title": "Declare JavaScript Objects as Variables",
       "description": [
         "Before we dive into Object Oriented Programming, let's revisit JavaScript objects.",
-        "Give your <code>motorBike</code> object a <code>wheels</code>, <code>engines</code> and <code>seats</code> attribute and set them to numbers."
+        "Give your <code>motorBike</code> object <code>wheels</code>, <code>engines</code> and <code>seats</code> attribute and set them to numbers."
```
Comment (erictleung) [inline seed/challenges/01-front-end-development-certification/object-oriented-and-functional-programming.json]: This line still needs a change.

```diff
- <code>seats</code> attribute and set them to numbers.
+ <code>seats</code> attributes and set each to a number.
```

**Review by erictleung — CHANGES_REQUESTED** [review]

**Comment by erictleung** [comment]
@91aman thanks for the PR! Almost there, just need to change the line I've pointed out just a little bit. Let us know in the [contributor's chat room](https://gitter.im/FreeCodeCamp/Contributors) if you need any help.

**Comment by 91aman** [comment]
@erictleung I have done the changes as requested by you. Do let me know if any others changes need to be done.

**Review by erictleung — APPROVED** [review]
LGTM! 👍

**Comment by 91aman** [comment]
@erictleung Thanks. what are next steps I need to do to get this PR merged?

**Comment by erictleung** [comment]
@91aman ah I was waiting for possibly others to take a look at this PR as well. But this kind of change should be safe to merge. I'll go ahead and merge it. Thanks for contributing! 🎉

**Comment by 91aman** [comment]
@erictleung Awesome thanks :)

**Pull request merged**